api/build.gradle

```diff
@@ -23,8 +24,6 @@ buildscript {
         SWAGGER_3_CODEGEN_VERSION = '3.0.34'
     }
 
-    project.ext['hibernate.version'] = '5.4.31.Final'
```

Comment:
I couldn't find anywhere this was used outside of this file, so I changed it to use our standard pattern.